stats/internal/record.go

```diff
@@ -21,5 +21,11 @@ import (
 // DefaultRecorder will be called for each Record call.
 var DefaultRecorder func(tags *tag.Map, measurement interface{}, attachments map[string]interface{})
 
+// MeasurementRecorder will be called for each Record call. This is the same as DefaultRecorder but
+// avoids interface{} conversion.
+// This will be a func(tags *tag.Map, measurement []Measurement, attachments map[string]interface{}) type,
+// but is interface{} here to avoid import loops
+var MeasurementRecorder interface{}
```

Comment:
If someone overrode this with a `func(tags *tag.Map, measurement []Measurement, attachments map[string]interface{})`, would `recorder := internal.MeasurementRecorder.(measurementRecorder)` accept it?

At first glance, this seems unnecessary, since this is already in an internal package

Reply:
Not 100% sure what you mean:

if you mean modified this L28 to use that instead of `interface{}` it would not compile (import loop)

If you mean set the value; we do (`internal.MeasurementRecorder = recordMeasurement`) but it can only be done in this package since its in `internal`, so a user couldn't do it?

Reply:
Oh, I see the import loop now.  This approach is probably the best we can do.  Are the benchmarks in the description updated for the latest attempt?

Reply:
I just updated it, but its the same (besides recording noise) as the initial iteration.